full-service/src/json_rpc/v2/api/request.rs

```diff
@@ -57,6 +57,7 @@ pub enum JsonCommandRequest {
         block_version: Option<String>,
         sender_memo_credential_subaddress_index: Option<String>,
         payment_request_id: Option<String>,
+        spend_only_from_subaddress: Option<String>,
```

Comment:
Updated all the APIs for building transactions to include the `spend_only_from_subaddress` filter